src/main.rs

```diff
@@ -39,23 +41,49 @@ struct Args {
     #[arg(long)]
     timeout_sec: u64,
 
+    // How long to cool off before runs
+    #[arg(long, default_value_t = 0)]
+    cooloff_sec: u64,
```

Comment:
I am not sure what real improvement will this provide, but can you provide a default value for it or have it as optional to not break the actual runner if not provided.

Reply:
It has a default value, right? And the default value makes it so that no behaviour changes.

Reply:
Yes, we already agreed on all of that.